org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java

```diff
@@ -983,10 +983,7 @@
 		private final Keyword cSubtitleSubtitleKeyword_3_0 = (Keyword)cSubtitleEnumLiteralDeclaration_3.eContents().get(0);
 		
 		//enum CellType:
-		//	default='Default' |
-		//	value1='Value1' |
-		//	value2='Value2' |
-		//	subtitle='Subtitle';
+		//	default='Default' | value1='Value1' | value2='Value2' | subtitle='Subtitle';
```

Comment:
this one is unrelated to the change, but caused by regeneration. leave it?

Reply:
yes keep